grub-btrfsd.initd

```diff
@@ -4,6 +4,7 @@
 
 name="grub-btrfs daemon"
 command="/usr/bin/grub-btrfs-openrc"
+command_args="${snapshots}"
 pidfile="/run/{RC_SVCNAME}.pid"
```

Comment:
Shouldn't this be `${RC_SVCNAME}` instead?

Reply:
No, because I changed `grub-btrfs-openrc` in order to take only one argument, which is the path to the snapshots directory. This is defined in the `snapshots` variable in the init script config file.